avro/tests/io.rs

```diff
@@ -25,7 +25,8 @@ fn schemas_to_validate() -> &'static Vec<(&'static str, Value)> {
     static SCHEMAS_TO_VALIDATE_ONCE: OnceLock<Vec<(&'static str, Value)>> = OnceLock::new();
     SCHEMAS_TO_VALIDATE_ONCE.get_or_init(|| {
         vec![
-            (r#""null""#, Value::Null),
+            // TODO: Do we want to support this? It makes the implementation significantly more complex
+            // (r#""null""#, Value::Null),
```

Comment:
What is the problem with it?
I am not sure whether it is really needed by someone. It was here when I adopted the codebase.